Script/Selenops.swift

```diff
@@ -81,5 +81,36 @@ func parse(document: String, url: URL) {
   collectLinks().forEach { pagesToVisit.insert($0) }
 }
 
+let args: [String] = CommandLine.arguments
+//lets check if they just use type -help or help
+if (args.count == 2 || args.count > 4) {
+  print("usage: swift selenops [startUrl searchWord [maxNumberOfPagesToVisit]]")
+  print("\t-Either no arguments can be used, two arguments can be used (startUrl")
+  print("\t and searchWord), or all three arguments can be used.")
+  exit(0)
+}
+//we should have a URL and a word
+if (args.count >= 3) {     //validate and set the word
```

Comment:
Brackets are not necessary (same [here](https://github.com/zntfdr/Selenops/pull/3/files#diff-494a3ac3c6af0696f0c2c017434d480eR104) & [here](https://github.com/zntfdr/Selenops/pull/3/files#diff-494a3ac3c6af0696f0c2c017434d480eR86))